app/bootstrap.php

```diff
@@ -63,14 +64,15 @@ function (ConsoleCommandEvent $event) use ($userInteraction) {
 };
 
 $container['installer.docker'] = function ($c) {
+    $dinghyCli = new DinghyCli($c['process.interactive_runner']);
```

Comment:
Yes, I think if a silent process runner is needed, we can just create a new service